fsw/src/hs_events.h

```diff
@@ -320,7 +320,7 @@
  *  This event message is issued when a reset counters command has
  *  been received.
  */
-#define HS_RESET_DBG_EID 24
+#define HS_RESET_INF_EID 24
```

Comment:
Why does the event name change from DBG to INF? Shouldn't line 316 also change to indicate Informational type?

Reply:
Yeah it's really a matter of interpretation whether to list some of these as DBG or INF.
I will update the comments to align with the common Event ID type as per the updates.

Reply:
Should be updated now mate.